manifests/0000_70_cluster-network-operator_04_deployment.yaml

```diff
@@ -62,6 +65,12 @@ spec:
         - name: host-kubeconfig
           hostPath:
             path: /etc/kubernetes/kubeconfig
+        - name: trusted-ca
+          configMap:
+            name: trusted-ca
+            optional: true
+            items:
+            - key: ca-bundle.crt
```

Comment:
i don't think this belongs in this PR.  This PR should be constrained to the job of taking the user provided CAs, merging them with the system CAs, and publishing that merged content into the openshift-config-managed configmap.

it should not delve into unrelated aspects like ensuring the network operator is mounting/consuming the proxy CA bundle

Reply:
ack, I'll move the operator ca mounting to a separate PR.